package.json

```diff
@@ -1,28 +1,30 @@
 {
-  "name": "seniorproject",
+  "name": "savis",
+  "productName": "SAVIS",
   "version": "1.0.0",
-  "description": "1. setup jekyll",
+  "description": "Statistical Analysis Vizualizions for Introtuctory Statistics",
   "main": "./_site/main.js",
-  "background": {
-    "script": "./_site/oneProportion/oneProportion.js"
-  },
   "dependencies": {
-    "chai": "^4.2.0",
-    "express": "^4.16.4",
-    "puppeteer": "^1.14.0"
+    "express": "^4.16.4"
   },
   "devDependencies": {
     "babel-core": "^6.26.3",
     "babel-plugin-transform-es2015-destructuring": "^6.23.0",
     "babel-plugin-transform-object-rest-spread": "^6.26.0",
     "babel-preset-es2015": "^6.24.1",
+    "chai": "^4.2.0",
+    "electron": "^4.1.4",
+    "electron-packager": "^13.1.1",
     "mocha": "^5.2.0",
-    "electron": "^4.1.4"
+    "puppeteer": "^1.14.0"
   },
   "scripts": {
     "test": "mocha --require babel-core/register",
     "test-all": "mocha --require babel-core/register --timeout=30000 test/**/*.test.js",
-    "start": "electron ."
+    "start": "electron .",
+    "package-mac": "electron-packager . --overwrite --platform=darwin --arch=x64 --prune=true --out=dist",
```

Comment:
I don't have a mac, can we test this on one? I can rent a laptop from the library tomorrow to test. Without knowing this library or having tried it yet, it seems strange that this one has no "savis" as an arg.